up42/initialization.py

```diff
@@ -54,6 +54,10 @@ def initialize_project(project_id: Optional[str] = None) -> "Project":
     Args:
         project_id: The UP42 project id
     """
+    warn(
```

Comment:
I think we should add the DeprecationWarning class as this is part of the same group of warnings (Applies to the other logs).

The stacklevel would be 1 by default, it means that the message will be emitted from the location where the `warn` was called. Do we want to emmit the warning from the method itself or from the Class?.

Reply:
ah totally forgot.
Will update.
Thanks!

Reply:
For initialization functions, the stack_level should be 1.
So I would skip the param `stacklevel=2`